### PR TITLE
Removing repeated words

### PR DESCRIPTION
**Pull request by puneet-ekline**
### Description
Updating the docs to remove repeated words.

More details [here](https://ekline.atlassian.net/wiki/external/MDJmNWI3MjBkMzE4NGQyMzhlYjkxNjUwNDVhYjBlOGQ)

### CODE REVIEW

**Review by theletterf — APPROVED** [review]
Thanks!

**Review by tsloughter — APPROVED** [review]

**Comment by theletterf** [comment]
@puneet-ekline Please run `npm run fix:format` from your CLI.

**Review by austinlparker — CHANGES_REQUESTED** [review]
see https://github.com/open-telemetry/opentelemetry.io/pull/5160#issuecomment-2336716134

**Review by cijothomas — APPROVED** [review]
LGTM

**Review by ferd — APPROVED** [review]

File: content/en/docs/collector/building/receiver.md [hunk 1]
```diff
@@ -520,8 +520,8 @@ func createTracesReceiver(_ context.Context, params receiver.CreateSettings, bas
 ```
 
 You now have all the necessary components to successfully instantiate your
-receiver factory using the `receiver.NewFactory` function. Go ahead and and
-update your `NewFactory()` function in your `factory.go` file as follow:
+receiver factory using the `receiver.NewFactory` function. Go ahead and update
+your `NewFactory()` function in your `factory.go` file as follow:
```
Comment (breedx-splk) [inline content/en/docs/collector/building/receiver.md:524]:
```suggestion
your `NewFactory()` function in your `factory.go` file as follows:
```

**Comment by theletterf** [comment]
@svrnm Shall we merge this one?

**Comment by svrnm** [comment]
> @svrnm Shall we merge this one?

Same here, let's finish the discussion first, see the other PR

**Comment by svrnm** [comment]
Please see https://github.com/open-telemetry/opentelemetry.io/pull/5160#issuecomment-2348281987

**Review by svrnm — CHANGES_REQUESTED** [review]
contentwise lgtm, I will block it with `request changes` to allow everyone to review the comment on our position on mixing a sales pitch / growth hacking with a contribution to our project.

File: content/en/docs/collector/building/receiver.md (hunk 1 above)
Comment (svrnm) [reply content/en/docs/collector/building/receiver.md:524]: @puneet-ekline please address this

**Review by svrnm — APPROVED** [review]

**Review by austinlparker — APPROVED** [review]

**Pull request merged**